docs/asus-rog-ryujin-guide.md

```diff
@@ -0,0 +1,97 @@
+# ASUS ROG RYUJIN II liquid cooler
+_Driver API and source code available in [`liquidctl.driver.asus_rog_ryujin`](../liquidctl/driver/asus_rog_ryujin.py)._
+
+_New in git._<br>
+
+## Initialization
+
+Initialization is not required. It outputs the firmware version:
+
+```
+# liquidctl initialize
+ASUS ROG RYUJIN II 360
+└── Firmware version    AURJ1-S750-0104
+```
+
+
+## Monitoring
+
+The cooler reports the liquid temperature, the speeds and duties of all fans.
+
+```
+# liquidctl status
+ASUS ROG RYUJIN II 360
+├── Liquid temperature                  26.2  °C
+├── Pump duty                             38  %
+├── Pump speed                          1410  rpm
+├── Embedded Micro Fan duty               40  %
+├── Embedded Micro Fan speed            2550  rpm
+├── AIO Fan Controller duty               38  %
+├── AIO Fan Controller speed - Fan 1     780  rpm
+├── AIO Fan Controller speed - Fan 2       0  rpm
+├── AIO Fan Controller speed - Fan 3       0  rpm
+└── AIO Fan Controller speed - Fan 4     750  rpm
+```
+
+
+## Speed control
+
+### Setting fan and embedded pump duty
+
+Pump duty can be set using channel `pump`.
+
+```
+# liquidctl set pump speed 90
+```
+
+Use channel `fans` to set all fans at the same time:
+
+```
+# liquidctl set fans speed 50
+```
+
+Use channel `fan1` to set the duty of the embedded fan:
+
+```
+# liquidctl set fan1 speed 50
+```
+
+Use channel `fan2` to set the duty of the fans connected to the AIO fan controller:
+
+```
+# liquidctl set fan2 speed 50
+```
```

Comment:
How about more meaningful fan channel names like `pump-fan` (for the embedded pump fan) and `external-fans` (for the external/fan controller/radiator fans)?